hub/api/tests/test_api.py

```diff
@@ -5,14 +5,35 @@
 from hub.core.tests.common import parametrize_all_dataset_storages
 
 
```

Comment:
is there a test for this with both memory cache & local storage?

Reply:
No. What case are you trying to cover?

Reply:
just to make sure this works in various configs.